frontend/public/locales/en/data-unavailable.json

```diff
@@ -3,6 +3,15 @@
   "breadcrumbs": {
     "unable-to-continue": "Unable to continue"
   },
-  "service-unavailable": "This service is not available to you. We did not find any record for your account.",
+  "service-eligiblle": "You are currently not a participant in the Canadian Dental Care Plan, to find out if you are eligible please visit Do you qualify on Canada.ca",
```

Comment:
**Nitpick:** 
```suggestion
  "service-eligible": "You are currently not a participant in the Canadian Dental Care Plan, to find out if you are eligible please visit Do you qualify on Canada.ca",
```

Same thing for French

Reply:
fixed